pkg/golinters/govet.go

```diff
@@ -59,20 +88,64 @@ func NewGovet(cfg *config.GovetSettings) *goanalysis.Linter {
 		unsafeptr.Analyzer,
 		unusedresult.Analyzer,
 	}
+}
+
+func isAnalyzerEnabled(name string, cfg *config.GovetSettings, defaultAnalyzers []*analysis.Analyzer) bool {
+	if cfg.EnableAll {
+		return true
+	}
+	// Raw for loops should be OK on small slice lengths.
+	for _, n := range cfg.Enable {
+		if n == name {
+			return true
+		}
+	}
+	for _, n := range cfg.Disable {
+		if n == name {
+			return true
```

Comment:
I guess it should be
```suggestion
			return false
```

Reply:
You are right, nice catch!
I've also added a corresponding test for that function.